cli/internal/packager/create.go

```diff
@@ -17,7 +17,10 @@ import (
 
 func Create(confirm bool) {
 
-	config.Load("zarf.yaml")
+	if err := config.LoadConfig("zarf.yaml"); err != nil {
```

Comment:
I don't think this is working as expected. I don't think the config is actually marshaling correctly. 

I noticed `make init-package` is failing because the zarf.yaml isn't recognized as an init package and is being saved off as `zarf-package-.tar.zst` instead of `zarf-init.tar.zst`. I did a tiny amount of poking around and I don't think the `var config ZarfConfig` is actually being populated here.